src/Pickles/Pickles/ObjectModel/Mapper.cs

```diff
@@ -46,33 +46,46 @@ public string MapToString(G.TableCell cell)
             return cell?.Value;
         }
 
-        public TableRow MapToTableRow(G.TableRow tableRow)
+        public TableRow MapToTableRow(G.TableRow tableRow, bool isForExample=false)
```

Comment:
The isForExample parameter is a code smell to me. You're steering the behavior of the class depending on this parameter. Is would prefer having a method MapToTableRow and another one MapToTableRowWithTestData and let the caller of the method decide which variant to use. The caller already makes that decision when they determine the value for isForExample, there is no need to make the same decision twice.